src/core/annotation.js

```diff
@@ -969,6 +969,37 @@ class Annotation {
     this.color = getRgbColor(color);
   }
 
+  /**
+   * Set the line ending; should only be used with FreeText annotations.
+   * @param {Name} lineEnding - The line ending name.
+   */
+  setLineEnding(lineEnding) {
```

Comment:
Please don't add an *almost* duplicate of an existing method, since both cases should use rather one common helper to reduce code-size.